src/components/status-button/status-button.css

```diff
@@ -1,4 +1,35 @@
 .StatusButton {
```

Comment:
This comment applies to all the css.

- The text should be displayed using upper case (check mock-up).
- Add the .StatusButton selector in all the CSS rules
- The button is not responsive when the parent container is smaller than the absolute with of the button.